.storybook/preview-head.html

```diff
@@ -1,8 +1,31 @@
-<style>
-    @font-face {
-      font-family: 'GT Super Text';
-      font-style: normal;
-      font-weight: 400;
-      src: url('fonts/GT-Super/GT-Super-Text-Medium.ttf') format('truetype');
-    }
+<style type="text/css">
```

Comment:
so we using local fonts?

Reply:
This is for storybook to display the font correctly